src/ChatEngine/ChatFeed/ChatHeader/props.tsx

```diff
@@ -1,8 +1,16 @@
 import { ChatHeaderStyles } from './styles';
 
 export interface Props {
+
+  /** id of the chat */
   id?: string | number;
+
+  /** title of the chat */
   title?: string;
+
+  /** description to be displayed as a subtitle */
   description?: string;
+
+  /** pass in a style object to override the default styles */
   customStyle?: ChatHeaderStyles;
 }
```

Comment:
Curious what the angle is here.
Are you recommending we add these comments every-time?
Great code shouldn't need comments, I'm okay with doing it while we establish policies/principles though

Reply:
https://tsdoc.org/

ya sorry this is to get it to show up in the args table, we can dial it back more just to show off the feature

butttt I would say this will be most useful for describing the styles and what each sub object does